rsts/concepts/admin.rst

```diff
@@ -104,7 +107,7 @@ For the sandbox development, no-op implementations of the notifications and sche
 Common
 ------
 
-As the name implies, ``common`` houses shared components used across different flyteadmin components in a single, top-level directory to avoid cyclic dependencies. These components include execution naming and phase utils, query filter definitions, query sorting definitions, and named constants.
+As the name implies, ``common`` houses shared components used across different FlyteAdmin components in a single, top-level directory to avoid cyclic dependencies. These components include execution naming and phase utils, query filter definitions, query sorting definitions, and named constants.
```

Comment:
```suggestion
As the name implies, ``common`` houses shared components used across different Flyte Admin components in a single, top-level directory to avoid cyclic dependencies. These components include execution naming and phase utils, query filter definitions, query sorting definitions, and named constants.
```